lib/ace/lib/event.js

```diff
@@ -115,6 +115,29 @@ exports.capture = function(el, eventHandler, releaseCaptureHandler) {
     return onMouseUp;
 };
 
+exports.addTouchMoveListener = function (el, callback) {
+    if ("ontouchmove" in el) {
+        var startx, starty;
+        exports.addListener(el, "touchstart", function (e) {
+            var touchObj = e.changedTouches[0];
+            startx = parseInt(touchObj.clientX);
+            starty = parseInt(touchObj.clientY);
+        });
+        exports.addListener(el, "touchmove", function (e) {
+            var factor = 1,
+            touchObj = e.changedTouches[0];
+
+            e.wheelX = -(parseInt(touchObj.clientX) - startx) / factor;
```

Comment:
From the W3 site,  
"clientX of type long, readonly  
The horizontal coordinate of point relative to the viewport in pixels, excluding any scroll offset"
Since it's a long, we want to parse it as an int to keep scrolling at individual pixel sizes. And thanks, I'll get right on that cla.